crates/core/Cargo.toml

```diff
@@ -13,7 +13,7 @@ readme = "README.md"
 [dependencies]
 bitcoin = { version = "0.32", default-features = false }
 serde = { version = "1", optional = true, features = ["derive", "rc"] }
-hashbrown = { version = "0.9.1", optional = true }
+hashbrown = { version = "0.15", optional = true }
```

Comment:
Looks like hashbrown 0.15.1 bumped the MSRV to 1.65 😞 . Are you OK with making this a pinned version?
```suggestion
hashbrown = { version = "=0.15.0", optional = true }
```